dashboards/charts/analytics.md

```diff
@@ -10,13 +10,19 @@ Forest Admin’s dashboard is handy when it comes to monitoring the overall KPIs
 “Analytics per record” are only supported in the API and Query modes.
 {% endhint %}
 
-* For Query charts, use the character `?` to inject the current record the user is currently seeing to your query.
+* For Query charts, in agent v1, use the character `?` to inject the current record the user is currently seeing to your query. Meanwhile in agent v2, you must use `{{recordId}}`.
```

Comment:
Maybe you can mention the feature name: templating. there is no page doc dedicated to templating but it is shared across other features (workspace, filters...)